src/content/i18n/fr.yml

```diff
@@ -1,14 +1,11 @@
 a11y.sectionLink: Titre de la section
 # Site settings
 site.title: Documentation Astro
-site.og.imageSrc: /default-og-image.png?v=1
-site.og.imageAlt: "Logo dAstro dans l'espace, avec une planète violette dans le style de saturne flottant à droite de limage."
```

Comment:
Ah damn, looking at this line I wonder if I carelessly deleted `'` quotes in this file or if I just messed up this line. Noticing “dAstro” and “limage” although “l'espace” survived.

Maybe we got lucky though because I haven’t immediately spotted any others in keys we’re keeping.

Reply:
Ah yes I had seen that at the time of my previous comment, I did not think to check in this PR if it happened elsewhere...
Apparently, there is only one other string concerned! (in the VF, I did not check the other languages 😅 )